projects/Mallard/src/services/logging.ts

```diff
@@ -153,7 +160,11 @@ class Logging {
 
     async clearLogs() {
         try {
-            return await loggingQueueCache.reset()
+            // Assumes there is a problem sending logs and clears them
+            const { isConnected } = await NetInfo.fetch()
+            if (isConnected) {
```

Comment:
We added this check because the app tries to clear logs on launch. 

Now that we have `maxAttempt` logic, do we still need this check? Do we still need to call `clearLog` from `Apps.tsx`?

Reply:
It's less likely to be needed but helps keep things clean. Could be removed to help keep things clean. Thoughts?

Reply:
As we discussed, keep things simple would be better.